### PR TITLE
Add support for Plesk API

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by CapsuleCD** [comment]
Hi.

I'm an automated pull request bot named [CapsuleCD](https://www.capsulecd.com). I handle testing, versioning and package releases for this project. 
- If you're the owner of this repo, you can click the button below to kick off the CapsuleCD build pipeline and create an automated release.'
- If not, don't worry, someone will be by shortly to check on your pull request. 

[CapsuleCD](https://www.capsulecd.com/project/github/AnalogJ/lexicon/pullrequests/278)

---
If you're interested in learning more about [CapsuleCD](http://www.github.com/AnalogJ/capsulecd), or adding it to your project, you can check it out [here](https://www.capsulecd.com)

**Comment by ctron** [comment]
Is there any chance to figure out what went wrong with CapsuleCD? It looks like circleci did run properly.

**Comment by AnalogJ** [comment]
Yeah, I'm working on fixing the capsulecd issue right now.

**Comment by AnalogJ** [comment]
Hey @ctron 
I finally got the automation working for packaging up & releasing `lexicon`

While attempting to release your PR, I did notice that you had added a new dependency `xmltodict`.
That's fine, however it needs to be wrapped in a try catch block so that its not a required dependency (only an optional dependency).

see https://github.com/AnalogJ/lexicon/blob/master/lexicon/providers/route53.py#L9-L13

Can you update the PR?

File: lexicon/providers/plesk.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
+import logging
+
+import requests
+import xmltodict
```
Comment (AnalogJ) [inline lexicon/providers/plesk.py]: add try catch around xmltodict import

similar to: https://github.com/AnalogJ/lexicon/blob/master/lexicon/providers/route53.py#L9-L13 and other optional deps.

**Comment by AnalogJ** [comment]
sorry about that, I've triggered a new release.

**Pull request merged**